srcpkgs/batsignal/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.1.2
+revision=1
+archs=noarch
+build_style=gnu-makefile
+makedepends=" libnotify-devel make pkg-config glib-devel "
```

Comment:
No need for a space after or before the quotes

Reply:
Sorry first time using git to pull request , didn't know that function existed ! 
I am going to take care of that and will squash the commits this time :)